### PR TITLE
Attempt to provide paths to missing includes

### PR DESCRIPTION
**Pull request by trelau**
- Still cleanup to do
- Need to make finding include directories more robust
- Remove all_includes.h since that is now auto generated

### CODE REVIEW

**Comment by trelau** [comment]
will eventually move the extra OCCT header files to pyOCCT. also a lot of reformatting was done by pycharm. include paths are essentially hard-coded...so.....not sure what to expect for now cross platform 😬

**Comment by trelau** [comment]
@frmdstryr worth a shot?

**Comment by frmdstryr** [comment]
:+1:

File: generate/run.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 from os.path import join, abspath, exists
 
+from binder.core import Generator
+
 # If running outside this folder we need to add this to the syspath
 BINDER_ROOT = os.path.dirname(os.path.dirname(__file__))
 if BINDER_ROOT not in sys.path:
     sys.path.append(BINDER_ROOT)
 
-from binder.core import Generator
```
Comment (frmdstryr) [inline generate/run.py:12]: I think pycharm re-ordering the imports here is why it's failing now. 

Modifying the sys path is a hack anyways so maybe there's a better way to do this.

**Comment by trelau** [comment]
yes i just moved it back in master. lots of cleanup to do later 😄

File: generate/run.py [hunk 2]
```diff
@@ -41,6 +43,22 @@ def find_clang_include_dir():
...
+    for path in get_search_paths():
+        vtk_include_path = join(path, 'include', 'vtk-8.2')
+        if exists(vtk_include_path):
+            print("Found {}".format(vtk_include_path))
+            return vtk_include_path
+
+
+def find_tbb_include_dir():
+    for path in get_search_paths():
+        tbb_include_path = join(path, 'Library', 'include')
```
Comment (frmdstryr) [inline generate/run.py:56]: I think it's the 'Library' here

Comment (trelau) [reply generate/run.py:56]: ya it needed that on windows, but prob not other platforms. just pushed a few more hacks, will try and rebuild and then assess

**Pull request merged**